tests/baselines/reference/derivedClassSuperCallsInNonConstructorMembers.errors.txt

```diff
@@ -49,97 +24,47 @@ tests/cases/conformance/classes/constructorDeclarations/superCalls/derivedClassS
            ~~~~~
 !!! error TS2304: Cannot find name 'super'.
                 ~
-!!! error TS1005: ';' expected.
+!!! error TS1441: Function call is not a type annotation.

```

Comment:
I find this one kind of strange. I think the thing that's strange to me is probably in part because the error span is limited to the `(`.

Reply:
Yeah the wordsmithing here stumped me 😕 ... Perhaps something around _"Cannot start a function call in a type annotation"_?

I would have liked to expand the error range to the full function call but didn't see a good way around doing that in the case of a big call with parameters.

Reply:
I'll go with that for now, it seems a little better than the current message.